openwpm/Extension/webext-instrumentation/src/lib/http-post-parser.ts

```diff
@@ -70,367 +41,6 @@ export class HttpPostParser {
         ),
       };
     }
-
-    // Return empty response until we have all instrumentation converted
```

Comment:
Do you know what this refers to?

Reply:
I think this was just precautionary since the previous file did a lot of parsing that's no longer necessary. We currently cover all properties of the requestBody. See [MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/onBeforeRequest#details).